src/components/utils/caseUtils.ts

```diff
@@ -225,7 +231,13 @@ export function useDownloadUtils() {
             setAbortController(controller);
 
             for (const c of cases) {
-                await exportCase(c, format, formatParameters, controller);
+                await exportCase(
+                    c,
+                    format,
+                    formatParameters,
+                    controller,
+                    fileName
```

Comment:
In my opinion, passing the same fileName for every cases is not a good idea, in fact the fileName you pass is only the fileName of the first element.

I created a suggestion PR here which uses a map between case uuid and file name for each case: https://github.com/gridsuite/gridexplore-app/pull/489

In this PR, I have corrected also comments mentioned in this PR..

Reply:
Merged. Everything seems to be fine.